docs/RELEASING.md

```diff
@@ -0,0 +1,16 @@
+Releasing
+=========
+
+- Using GitHub create a new releasing branch from `next`: `releases/v<version number>`
+- Checkout the release branch
+  - Bump the version number: `make VERSION=<version> bump`
+  - Inspect the updated CHANGELOG, README, and version files to ensure they are correct
+  - Open a PR from the release branch to `main`
+- Once merged:
+  - Pull the latest changes into a *fresh* local repository (clone `main`)
```

Comment:
Is this always required, or can we rely on `make clean`?

Reply:
We can't completely rely on `make clean` as far as I've seen.

Reply:
I thought that was the idea of #102? What about running `git clean`?

Reply:
That is the idea of #102 but it is very easy for new files / dirs to slip in and accidentally be included in the release. I think a `git clean` would also be an acceptable solution. I'll amend the doc